packages/mask/shared-ui/locales/en-US.json

```diff
@@ -1083,6 +1083,18 @@
     "plugin_artblocks_smt_wrong": "Something went wrong!",
     "plugin_artblocks_share": "I just purchased a beautiful piece of art from '{{name}}' collection for {{price}} {{symbol}}. Install @realMaskNetwork to get yours.",
     "plugin_artblocks_share_no_official_account": "I just purchased a beautiful piece of art from '{{name}}' collection for {{price}} {{symbol}}. Welcome to join.",
+    "plugin_tips_disconnect_intro": "Are you going to disconnect {{persona}} from wallet <strong>{{address}}<strong/>?",
+    "plugin_tips_disconnect_dialog_title": "Disconnect Wallet ?",
```

Comment:
```suggestion
    "plugin_tips_disconnect_dialog_title": "Disconnect Wallet?",
```

Reply:
Weirdly, a dialog title ends in a question mark?